pkg/cmd/projectCreate.go

```diff
@@ -660,6 +709,48 @@ func createTestFolderGradle(fileGenerator *utils.FileGenerator, targetTestFolder
 	return err
 }
 
+// Creates a build.gradle and a bnd.bnd file in a Gradle test project directory.
+func createManagerFolder(fileGenerator *utils.FileGenerator, targetTestFolderPath string,
+	packageName string, featureName string, forceOverwrite bool, isDevelopment bool) error {
+
+	type TestGradleParameters struct {
```

Comment:
The rest of this code is trying to create a file using a template, so should be in it's own method ?

Reply:
Yes, I do understand there is not any of this method. My apologies, it's been added mistakenly.